common/lib/xmodule/xmodule/modulestore/tests/test_mongo_call_count_import_export.py

```diff
@@ -0,0 +1,119 @@
+"""
+Tests to verify correct number of MongoDB calls during course import/export and traversal
+when using the Split modulestore.
```

Comment:
Since this test file is specific to testing Split, do you want to rename the file to include "split"?